app/views/pages/medicaid/intro_household.erb

```diff
@@ -0,0 +1,33 @@
+<h4 class="step-header step-header__title--centered">Introduction</h4>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Now tell us about any other people in your household.</div>
+    <p class="text--help text--centered">
+      This includes everyone who lives in your home (including you), even if they are not seeking benefits today.
+    </p>
+  </header>
+
+  <div class="form-card__content">
+    <div class="slab slab--not-padded">
+      <div class="card card--narrow">
+        <p>
+          <i class="button__icon--left icon-check icon-check--color"></i>
+          Just me!
+        </p>
+
+        <a href="/pages/medicaid/intro_household_member" class="button button-full-width button-bold">
+          <i class="button__icon--left icon-add"></i>
+          Add a member
+        </a>
+
+      </div>
+    </div>
+  </div>
+
+  <footer class="form-card__button_right">
+    <a href="" class="button button--nav button--nav-center button--cta button--full-width">
```

Comment:
I *think* that setting the href value to `"#"` is perhaps better? Maybe it doesn't really matter...